packages/types/src/interfaces/extrinsic/v1/Extrinsic.ts

```diff
@@ -95,8 +96,9 @@ export default class CENNZnetExtrinsic extends GenericExtrinsic{
     const cennznetAddress = cvmToAddress(ethAddress);
     const nonce = await api.rpc.system.accountNextIndex(cennznetAddress);
     const payload = this.registry.createType('EthWalletCall', { call: this, nonce }).toHex();
+    const encodedPayload = `data:application-octet;base64,${base64Encode(payload)}`;
```

Comment:
should be `data:application/octet-stream`

Reply:
ya was waiting for image from node changes.. updated prefix and signature with change in prefix